src/main/java/com/cloudbees/jenkins/plugins/bitbucket/api/BitbucketOAuthService.java

```diff
@@ -0,0 +1,46 @@
+package com.cloudbees.jenkins.plugins.bitbucket.api;
+
+import org.eclipse.jgit.util.Base64;
```

Comment:
This is not base64 from commons codec.

Reply:
yes, but it uses base32 from commons
btw, test passed